duo_wordpress.php

```diff
@@ -378,6 +383,15 @@ function duo_settings_xmlrpc() {
         echo "<input id='duo_xmlrpc' name='duo_xmlrpc' type='checkbox' value='off' $val /> Yes<br />";
         echo "Using XML-RPC bypasses two-factor authentication and makes your website less secure. We recommend only using the WordPress web interface for managing your WordPress website.";
     }
+    function duo_settings_rest() {
+        $val = '';
+        if(duo_get_option('duo_rest', 'off') == 'off') {
```

Comment:
@JohnMaguire I'm following the pattern of the current `duo_settings_xmlrpc` here .. see https://github.com/duosecurity/duo_wordpress/blob/master/duo_wordpress.php#L375 ... that is, shouldn't disable for XMLRPC be disabled by default too?